validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -218,6 +218,13 @@ public static ValidatorClientService create(
             () -> validatorClientService.initializeValidators(validatorApiChannel, asyncRunner))
         .thenCompose(
             __ -> {
+              if (validatorConfig.isExitWhenNoValidatorKeysEnabled()
+                  && validatorLoader.getOwnedValidators().getActiveValidators().isEmpty()) {
+                LOG.info(
```

Comment:
Maybe error? Also this message is not displayed in console if console log option is not on. Maybe use StatusLogger?

Reply:
humm I was doing the same as `DoppelgangerDetectionShutDown`.. maybe we could migrate everithing to status logger. Not sure it is an error though...